Platformsh.php

```diff
@@ -222,19 +229,22 @@ protected function installMagento()
             --db-host=$this->dbHost \
             --db-name=$this->dbName \
             --db-user=$this->dbUser \
-            --backend-frontname=$this->adminUrl \
-            --admin-user=$this->adminUsername \
-            --admin-firstname=$this->adminFirstname \
-            --admin-lastname=$this->adminLastname \
-            --admin-email=$this->adminEmail \
-            --admin-password=$this->adminPassword";
-
+            --backend-frontname={$this->adminVariables['ADMIN_URL']} \
+            --admin-user={$this->adminVariables['ADMIN_USERNAME']} \
```

Comment:
It would also be great if shell argument escaping would be applied here. http://php.net/escapeshellarg